extra/qemu/PKGBUILD

```diff
@@ -23,15 +23,15 @@ pkgname=(
   qemu-tests
   qemu-tools
   qemu-ui-{curses,dbus,egl-headless,gtk,opengl,sdl,spice-{app,core}}
-  qemu-user{,-static}{,-binfmt}
+  #qemu-user{,-static}{,-binfmt}
   qemu-vhost-user-gpu
   qemu-virtiofsd
   qemu-{base,desktop,emulators-full,full}
 )
 pkgver=7.1.0
-pkgrel=10
+pkgrel=11
 pkgdesc="A generic and open source machine emulator and virtualizer"
-arch=(x86_64)
+arch=(x86_64 aarch64)
```

Comment:
😬

Reply:
What's the problem?

I'm building the whole qemu group natively thus adding the arch.

Reply:
Sorry, this is more me being uncomfortable that the archlinuxarm PKGBUILD for qemu was copied over without even changing the arch, which makes me wonder what other packages are configured incorrectly like this. Does the builder just ignore the arch?

Reply:
Oh, I see. The Alarm modifies its build systems to mostly ignore the `arch` specification.

I just hate the extra warning when running "makepkg".

You can ignore the change completely and accept the warning.